src/Arcus.Testing.Tests.Integration/Arcus.Testing.Tests.Integration.csproj

```diff
@@ -17,7 +17,7 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="8.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
-    <PackageReference Include="xunit" Version="2.3.1" />
+    <PackageReference Include="xunit" Version="2.9.2" />
```

Comment:
Had to update this to use the new `Assert.Contains` functionality that includes an overload with a `IDictionary<,>`. This was needed to assert whether a table entity contained a certain key/value-pair.